### ISSUE TITLE
[TOPIC-REQUEST] Parallel algorithms

### ISSUE THREAD

**Issue by diehlpk** [cplusplus/SG20#94]
I taught parallel algorithms in my course and I think it would be great to add this topic here. 

I would be willing to prepare a draft for this topic.

**Comment by vulder** [comment]
From our discussion in the last SG20: Sounds like a good and worth while topic. :+1:. We would like to encourage you to contribute a new topic on this.
In which module would see this topic?
PS: A suggestion would be to introduce a separate module for parallel programming, would that be a good fit?

**Comment by diehlpk** [comment]
Yes, having a new separate module for parallel programming would be good, we could add parallel algorithms, async programming, and sender and receivers, there.

**Comment by vulder** [comment]
Ok, sounds good. As SG20 already agreed upon the topic in the last meeting, I'm switching this request into a issue to create a first draft of the topic.

**Comment by diehlpk** [comment]
Should we have a topic for algorithms as well?

**Comment by vulder** [comment]
We already have on listen in p1725, maybe the issue is missing on gh. I'll make a pass on the weekend and double check.

**Comment by diehlpk** [comment]
@vulder Please add the NG label
